docker-compose-services/elasticsearch/docker-compose.elasticsearch.yaml

```diff
@@ -15,6 +15,12 @@ services:
       - VIRTUAL_HOST=$DDEV_HOSTNAME
       - HTTP_EXPOSE=9200:9200
       - HTTPS_EXPOSE=9201:9200
+
+      # Allow CORS.
+      - http.cors.enabled=true
```

Comment:
These are really lower-case environment variables named like `http.cors.enabled` ???
And what does this addition have to do with the stated goal of adding SSL info?
And what process consumes these environment variables?

Reply:
Right, might have needed a different PR - sorry. 

My use case is that I'm calling ES directly from client side via JS. That didn't work, due to CORS errors until I've added those environment variables?

Reply:
Client side where? Those variables get injected into the web container... so if they're going to affect something it's going to be nginx? Anyway, unless we know for sure why we're adding them, we can't have them in the PR.

Reply:
My use case is that my client side app was calling ES directly, so I needed to enable CORS

![image](https://user-images.githubusercontent.com/125707/78766717-4bac7180-7992-11ea-8f77-1b096463374d.png)

I found that config [here](https://github.com/appbaseio/dejavu/issues/256) -- and it worked for me.

So I'll remove this from this PR, question is if you'd want it in another PR -- maybe have that code commented out, just to save others the time to find this?

Reply:
Makes sense, but we need to know why. And yes, let's get it into a different PR. I guess those must change the behavior of the java webserver elasticsearch is using. It would probably be better to incorporate them into launch args for elasticsearch, but I'm OK with this approach if we know why we're doing it. Must be in the elasticsearch docs.